includes/wcs-renewal-functions.php

```diff
@@ -143,6 +152,35 @@ function wcs_get_last_non_early_renewal_order( $subscription ) {
 	return $last_non_early_renewal;
 }
 
+/**
+ * Get the last renewal order (early renewals included).
+ *
+ * @param WC_Subscription $subscription The subscription object.
+ * @return WC_Order|bool The last non-early renewal order, otherwise false.
+ */
+function wcs_get_last_renewal_order( $subscription ) {
+	$renewal_orders = wcs_get_renewal_orders_sorted_by( $subscription, 'date_created' );
+	return $renewal_orders ? reset( $renewal_orders ) : false;
+}
+
+/**
+ * Gets the renewal orders for a subscription, sorted by the specified property.
+ *
+ * @param WC_Subscription $subscription The subscription object.
+ * @param string          $sort_by      The subscription property to sort by.
+ * @param string          $order        Optional. The sort order to sort by. Default is 'descending'.
+ *
+ * @return WC_Order[] The subscriptions renewal orders sorted.
+ */
+function wcs_get_renewal_orders_sorted_by( $subscription, $sort_by, $order = 'descending' ) {
+	$renewal_orders = $subscription->get_related_orders( 'all', 'renewal' );
+
+	// We need the orders sorted by the date they were created, with the newest first.
```

Comment:
```suggestion
	
```

Now that this function is generic this comment is no longer correct in all cases. I've suggested we just delete it but feel free to submit a replacement too.